src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figSup = new FigureSupplier();
+        for (int i = 0; i < 6; i++) {
+            if (i < 4) {
```

Comment:
```suggestion
            if (i < figures.length()) {
```

Reply:
in readme 3 figure by random and 3 by default... 
so i can use figures.length() in FOR and in IF for example figures.length() / 2

Reply:
yeah, sorry, I meant to suggest length/2